deployments/common/global-configuration.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: k8s.nginx.org/v1alpha1
```

Comment:
Do we need helm files as well?

Reply:
We do, but I would implement that separately